src/components/category.vue

```diff
@@ -110,22 +110,8 @@ export default {
 			} );
 		},
 		formatDate: function( value ) {
-
-			value = value.date;
-			if ( value ) {
-				const date = new Date( value );
-				const monthNames = [ "January", "February", "March",
-					"April", "May", "June", "July",
-					"August", "September", "October",
-					"November", "December" ];
-
-				const day = date.getDate();
-				const monthIndex = date.getMonth();
-				const year = date.getFullYear();
-
-				return monthNames[ monthIndex ] + ',' + day + ' ' + year;
-			}
-
+			const vm = this;
```

Comment:
I think Making file of filters and importing it in components would also lead to redundancy. Here Redundancy of unnecessary filters.